packages/expo-av/android/build.gradle

```diff
@@ -74,11 +74,9 @@ dependencies {
 
   // Newer version introduces dependency versions conflict
   // on 'com.android.support:support-annotations'
```

Comment:
```suggestion
```

It appears there isn't a conflict anymore, hence this PR is upgrading exoplayer.